components/strava/actions/create-activity/create-activity.mjs

```diff
@@ -0,0 +1,83 @@
+import strava from "../../strava.app.js";
+
+export default {
+  name: "Create Activity",
+  description: "Creates a manual activity for an athlete. See `https://developers.strava.com/docs/reference/`",
```

Comment:
Could you please use `[]()` to ref the docs?

Reply:
done